### ISSUE TITLE
"Copy Client Player" on Player Entity element doesn't work

### ISSUE THREAD

**Issue by Cnilssonak** [Keksuccino/FancyMenu#477]
TLDR; "Copy Client Player" option is pulling up the client player name but not the client skin or cape.

I'm using the 1.19.2 version, and I'm attempting to include the player entities of myself and my friend in my custom menu. For myself, I was trying to use the "Copy Client Player" option, but it's only replicating my username and not my skin or cape. I've checked my other mods, and none seem to be affecting it.
I hope the images help, and thank you for your time!

(As a side note, I'm wondering If there's anywhere I can find information on how to set skin textures)

<img width="1311" alt="Screenshot 2022-12-31 at 1 26 15 AM" src="https://user-images.githubusercontent.com/114829635/210133368-c344fa0f-02ec-4478-9de4-0c4d25536c55.png">
<img width="465" alt="Screenshot 2022-12-31 at 1 43 00 AM" src="https://user-images.githubusercontent.com/114829635/210133836-2e9d1478-a8b3-4fe0-9a4c-4434d494c425.png">
<img width="1004" alt="Screenshot 2022-12-31 at 1 41 18 AM" src="https://user-images.githubusercontent.com/114829635/210133802-3fa0d103-1324-4c8d-b468-47eeb2532056.png">

**Comment by Keksuccino** [comment]
Please send me your `logs/latest.log` file after trying to enable "Copy Client Player".

This is an issue that happens to some, but not all users and I have no clue why. It works fine for me.

**Comment by Keksuccino** [comment]
And just to be sure: Did you have an active internet connection when trying this? You need internet to get the skin and cape textures.

**Comment by Cnilssonak** [comment]
Yes, I was connected to an active internet connection. 
[latest.log](https://github.com/Keksuccino/FancyMenu/files/10341698/latest.log)

**Comment by Keksuccino** [comment]
Fixed/Added in FancyMenu v2.13.3!

**Comment by Cnilssonak** [comment]
Thanks!

<img width="1440" alt="Screenshot 2023-01-15 at 11 08 04 PM" src="https://user-images.githubusercontent.com/114829635/212628150-092d3af4-4f49-4080-acfe-7f12128aefd5.png">
